src/xmipp/bindings/python/xmipp_base.py

```diff
@@ -1,93 +1,105 @@
-
+import re
+from collections import OrderedDict
+from distutils.spawn import find_executable
 
 from xmippLib import *
 import os
 import sys
+import subprocess
+
 
 def xmippExists(path):
     return FileName(path).exists()
 
 def getXmippPath(*paths):
-    ''' Return the path of the Xmipp installation folder
+    """ Return the path of the Xmipp installation folder
         if a subfolder is provided, will be concatenated to the path
-    '''
-    if 'XMIPP_HOME' in os.environ:
-        return os.path.join(os.environ['XMIPP_HOME'], *paths)
-    else:
-        # xmipp   =     build      <   bindings    <     python    <     xmipp_base.py
-        xmippHome = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
-        if os.path.isfile(os.path.join(xmippHome, 'lib', 'libXmipp.so')):
-            print("Warning: XMIPP_HOME not found.\n"
-                  "Using an auto-detected directory: " + xmippHome)
+    """
+
+    candidates = []  # First candidate from XMIPP_HOME, second from this file path
+    envHome = os.environ.get('XMIPP_HOME', '')  # the join do nothing if second is absolute
+    candidates.append(os.path.join(os.environ.get('SCIPION_HOME', ''), envHome))
```

Comment:
each time we do this, some fluffy bunny dies somewhere in the most excruciating way possible

Reply:
XMIPP_HOME can be relative to SCIPION_HOME, we should manage it because in Xmipp we should know nothing about Scipion.